CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,50 @@
+
+All members share a serious responsibility for the community’s good public relations, especially at the community level. Their readiness to help with religious, charitable, educational, and civic activities brings credit to the community and is encouraged. 
+
+Members who have access to community funds in any form must follow the prescribed procedures for recording, handling, and protecting money as detailed in the community’s policies and procedures or other explanatory materials, or both. 
+
+When handling financial and personal information about members or others with whom the community has dealings, the following principles are to be abided: 
+- Collect, use, and retain only the personal information necessary for the community’s business. Whenever possible, obtain any relevant information directly from the person concerned. Use only reputable and reliable sources to supplement this information. 
+- Retain information only for as long as necessary or as required by law. Protect the physical security of this information. 
+- Limit internal access to personal information to those with a legitimate business reason for seeking that information. Only use personal information for the purposes for which it was originally obtained. Obtain the consent of the person concerned before externally disclosing any personal information, unless legal process or contractual obligation provides otherwise. 
+
+
+Unacceptable Behaviour:
+
+Members must not engage in
+Sexual harassment or conduct themselves in a way that could be construed as such, for example, by using inappropriate language, keeping or posting inappropriate materials in their work area, or accessing inappropriate materials on their computer.
+
+The community imposes strict standards to prevent fraud and dishonesty. If members become aware of any evidence of fraud and dishonesty, they should immediately inform higher authorities of the club. 
+
+Community funds and all other assets of the community are proposed for the community only and not for personal benefit. This includes the personal use of community based assets, such as computers. It is strictly prohibited.
+
+Members must take care to separate their personal roles from their community positions when communicating on matters not involving community business. members must not use community identification, stationery, supplies, and equipment for personal or political matters.
+
+When communicating publicly on matters that involve community business, members must not presume to speak for the community on any topic, unless they are certain that the views they express are those of the community, and it is the community’s desire that such views be publicly disseminated.
+
+When dealing with anyone outside the community, including public officials, members must take care not to compromise the integrity or damage the reputation of either the community, or any outside individual, business, or government body.
+
+How to report:
```

Comment:
Where is the core committee situated, how does a freshly onboarded individual know who's who!

Reply:
The current organisers of the club are tentatively the core committee. And they are all approachable to the members.